airbyte-webapp/src/components/JobItem/components/ResetStreamDetails.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+interface IProps {
+  names: string[];
+}
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 4px;
+`;
```

Comment:
We're trying to convert everything to SASS at the moment. Could you please use SASS modules for this newly created component.